baseline/reader.py

```diff
@@ -155,7 +155,11 @@ def __init__(self, vectorizers, trim=False, truncate=False):
         self.tgt_vectorizer = None
         for k, vectorizer in vectorizers.items():
             if k == 'tgt':
-                self.tgt_vectorizer = GOVectorizer(vectorizer)
+                self.tgt_vectorizer = vectorizer
+                if not self.tgt_vectorizer.emit_begin_tok:
+                    self.tgt_vectorizer.emit_begin_tok.append('<GO>')
```

Comment:
would be better if it was `Offsets.VALUES[Offsets.GO]]`

Reply:
done